src/olympia/editors/models.py

```diff
@@ -722,6 +722,24 @@ def calculate_verdict(
         return verdict_info
 
     @classmethod
+    def verdict_info_prettifier(cls, verdict_info):
+        """Return a generator of strings representing the a verdict_info
+        (as computed by calculate_verdict()) in human-readable form."""
+        mapping = {
+            'uses_custom_csp': ugettext(u'Uses a custom CSP'),
```

Comment:
0821e92ae422d9e7eb0aaa9c3bfed9af5aef011d (they used to be separated by commas before I made an HTML list, which is why the sentences were missing periods)